source/monitor/monitor_kraken/app.py

```diff
@@ -114,5 +122,107 @@ def monitor():
         sock.close()
 
 
+# Status of a single kraken through http://MONITOR_HOST/status
+# The kraken is accessed through the zmq socket defined in "KRAKEN_GENERAL_zmq_socket"
+# Performs a zmq status request to kraken
+# and dumps the response into json
+@app.route('/status')
+def status():
+    zmq_socket = os.environ.get("KRAKEN_GENERAL_zmq_socket")
+    if not zmq_socket:
+        return json.dumps({'error': ['no zmq socket configured']}), 500
+    zmq_timeout_in_ms = os.environ.get("MONITOR_KRAKEN_ZMQ_TIMEOUT", 10000)
```

Comment:
Should you add it to some kind of shared default conf (maybe just mutualize that value)?
Also 10s looks a bit long for timeout on a status, maybe devops would have an opinion on that.